.github/workflows/build_windows.yml

```
@@ -213,20 +213,21 @@ jobs:
-        LUPDATE_ARGS: ""
-        POSTPROCESS_ARGS: "--warn-only ${{ env.DO_PLACEHOLDER_TRANSLATIONS == 'true' && '--generate-placeholder-translations' || '' }}"
-        POSTPROCESS_LAUNCHER: "python3 -X utf8"
-      run: |
-        bash ./tools/translations/run_lupdate.sh
-    - name: Update .ts files (tx pull)
-      if: env.DO_UPDATE_TS == 'true'
-      uses: cbjeukendrup/cli-action@windows
-      with:
-        token: ${{ secrets.TRANSIFEX_API_TOKEN }}
-        args: "pull -f"
+    - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)